src/gui_common/CustomRichTextLabel.cs

```diff
@@ -225,32 +225,52 @@ private string BuildTemplateForTag(string input, ThriveBbCode bbcode, List<strin
         // Defaults to input so if something fails output returns unchanged
         var output = input;
 
+        var simulationParameters = SimulationParameters.Instance;
+
         switch (bbcode)
         {
             case ThriveBbCode.Compound:
             {
-                if (!SimulationParameters.Instance.DoesCompoundExist(input))
-                {
-                    GD.Print($"Compound: \"{input}\" doesn't exist, referenced in bbcode");
-                    break;
-                }
-
-                var compound = SimulationParameters.Instance.GetCompound(input);
-
-                var name = compound.Name;
-
                 var pairs = StringUtils.ParseKeyValuePairs(attributes);
 
-                // Parse attributes if there are any
-                if (pairs.TryGetValue("text", out string value))
+                var fallback = false;
```

Comment:
Could you add a comment and create an issue that this fallback should be removed, within like 6 months to give some time for translations to be updated?